src/components/cart/CartToggle/cart-toggle.scss

```diff
@@ -14,12 +14,12 @@
     justify-content: center;
     min-width: var(--cart-toggle-size);
     height: var(--cart-toggle-size);
-    padding: var(--space-0);
-    color: var(--color-text-inverse);
-    font-weight: var(--text-weight-bold);
-    font-size: var(--text-size-0);
-    background: var(--bg-secondary-default);
-    border-radius: var(--border-radius-pill);
+    padding: var(--fs-spacing-0);
+    color: var(--fs-color-text-inverse);
+    font-weight: var(--fs-text-weight-bold);
+    font-size: var(--fs-text-size-0);
+    background: var(--fs-color-primary-bkg);
```

Comment:
Isn't background also a type? Or is just a state 🤔

Reply:
Actually, no. The type is indicated by the prefixes '-color' and '-primary'; '-color' refers to the CSS property, while '-primary' refers to its hierarchical group; finally, '-bkg' refers to the actual property (category). Because this is a semantic phrasing arrangement, we must occasionally deviate from pure CSS syntax to make sense of these groupings.